01-node-tutorial/17-http-stream.js

```diff
@@ -1,16 +1,27 @@
 var http = require('http')
 var fs = require('fs')
-
+const { file } = require('loader')
```

Comment:
Requires changes:
This is not a valid required import. It's also not used and can be removed!